website/www/site/i18n/home/calendar/en.yaml

```diff
@@ -0,0 +1,42 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+ 
+- id: home-calendar-title
+  translation: "Stay up to date with Beam"
+- id: home-calendar-category-tag-first
+  translation: "Category"
+- id: home-calendar-category-date-first
+  translation: "2020/02/21"
+- id: home-calendar-card-title-first
+  translation: "Performance-Driven Runtime Type Checking for the Python SDK"
+- id: home-calendar-card-author-first
+  translation: "Saavan Nanavati"
+- id: home-calendar-category-tag-second
+  translation: "Category"
+- id: home-calendar-category-date-second
+  translation: "2020/02/21"
+- id: home-calendar-card-title-second
+  translation: "Performance-Driven Runtime Type Checking for the Python SDK"
+- id: home-calendar-card-author-second
+  translation: "Saavan Nanavati"
+- id: home-calendar-category-tag-third
+  translation: "Category"
+- id: home-calendar-category-date-third
+  translation: "2020/02/21"
+- id: home-calendar-card-title-third
+  translation: "Performance-Driven Runtime Type Checking for the Python SDK"
+- id: home-calendar-card-author-third
+  translation: "Saavan Nanavati"
+- id: home-calendar-card-events-title
+  translation: "Upcoming events"
+- id: home-calendar-card-events-button
+  translation: "SEE MORE"
```

Comment:
What is the difference between files like this (`i18n/**/en.yaml`), and the miscellaneous `en` directories in `data`? AFAICT the difference is just that i18n just has simpler id - translation while the config in data can be more complex and structured.

Based on that distinction it seems like much of this would be more appropriate in `data`

Reply:
Yeah. What I understand is, i18n directory is naturally offered by the Hugo plugin with the same name and it's used for localizing strings, syntax  `{{ i18n "id here" }}` or `{{ T "id here" }}`. First, I got stuck when I needed to use a loop structure, plus translation together. Then I figured out that `data` let me to apply a loop structure on it as an array, and i18n helped me to separate directories according to the main language (data/en, data/pl, data/sp, etc.). I know it's a bit annoying to have 2 places, but simply think that one for strings and one for arrays. 😛

Reply:
Gotcha. It seems like with that pattern home-calendar* items could be represented in data with a structure like:
```
home-calendar-first:
- category-tag: "Category"
- category-date: "2020/02/21"
- card-title: "Performance-Driven Runtime Type Checking for the Python SDK"
- card-author: "Saavan Nanavati"
```